feature-topic/src/main/java/com/google/samples/apps/nowinandroid/feature/topic/TopicScreen.kt

```diff
@@ -174,8 +172,6 @@ private fun LazyListScope.TopicCards(news: NewsUiState) {
             newsResourceCardItems(
                 items = news.news,
                 newsResourceMapper = { it },
-                isBookmarkedMapper = { /* TODO */ false },
```

Comment:
You shouldn't remove these, they're todo markers for future feature work.

Reply:
@tunjid Sure, I reverted these parts.